app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="card">
+  <div class="card-header">
+    <h2>Sign up</h2>
+  </div>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { multipart: true }) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field form-group">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, required: true %>
+    </div>
+
+    <div class="field form-group">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field form-group">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field form-group">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field form-group">
+      <%= f.label :profile_picture %><br />
+      <%= f.file_field :profile_picture %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
+  <% end %>
+  <div class="actions">
+    <%= render "devise/shared/links" %>
+  </div>
+</div>
```

Comment:
- [ ] Kindly improve the styling of the sign up page so that it matches the design.

  Your Implementation | Expected Design
  :---:|:---:
  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/08942a70-80e2-4b0b-8842-1041c09cff48) | ![image](https://github.com/Tracy-miranja/budget/assets/36542986/1988dff7-ff62-4777-8985-86a3f1671b90)